src/queries/dash/domain-list.sql

```diff
@@ -0,0 +1,93 @@
+--- description: List of domains along with some summary data.
+--- Access-Control-Allow-Origin: *
+--- Cache-Control: max-age=3600
+--- timezone: UTC
+--- device: all
+--- domainkey: secret
+
+WITH pvs AS (
+  SELECT
+    weight,
+    REGEXP_REPLACE(hostname, r'^www.', '') AS hostname,
+    COUNT(DISTINCT id) AS ids,
+    FORMAT_DATE('%Y-%b', time) AS month,
+    MIN(time) AS first_visit,
+    MAX(time) AS last_visit
+  FROM
+    helix_rum.EVENTS_V3(
```

Comment:
If you use `PAGEVIEWS_V3` you don't have to do the `SUM(weight * ids)` math yourself.

Reply:
This is done.  PAGEVIEWS_V3 is not as accurate since it takes MAX(weight) instead of proportionally applying different weights * ids but if we care enough to fix, that can be done within PAGEVIEWS_V3 itself.  I can also see that we should standardize across queries how we do this, for example the BDF ETL query does not currently use PAGEVIEWS_V3.